Makefile

```diff
@@ -357,8 +364,11 @@ DEFAULT_MANIFEST_SRC=./_site/$(RELEASE_STREAM)/getting-started/kubernetes/instal
 endif
 MANIFEST_SRC?=$(DEFAULT_MANIFEST_SRC)
 
-## Create an archive that contains a complete "Calico" release
-release-archive: release-prereqs $(RELEASE_DIR).tgz
+$(RELEASE_WINDOWS_ZIP):
+	wget https://github.com/projectcalico/node/releases/download/$(NODE_VER)/calico-windows-$(NODE_VER).zip -P $(OUTPUT_DIR)
+
+## Create an archive that contains a complete "Calico" release. This includes the release tarball (which bundles manifests, images, and binaries) and the Calico for Windows installation archive.
+release-archive: release-prereqs $(RELEASE_DIR).tgz $(RELEASE_WINDOWS_ZIP)
```

Comment:
Why don't we add the windows release to the full release archive? That's what we've done with all of the other artifacts that are part of a release.

Reply:
I did it this way because `install-calico-windows.ps1` downloads the Windows zip file from the Calico release.

Reply:
ah, gotcha.